h/security/__init__.py

```diff
@@ -41,4 +41,4 @@ def includeme(config):  # pragma: no cover
             "being available to ANYONE!"
         )
 
-    config.set_security_policy(TopLevelPolicy(proxy_auth=proxy_auth))
```

Comment:
No need to pass `proxy_auth` into `TopLevelPolicy()` here, the setting can be read directly where it's needed instead.